modules/core/src/main/scala/org/scalasteward/core/repoconfig/ValidateRepoConfigAlg.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2018-2022 Scala Steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scalasteward.core.repoconfig
+
+import better.files.File
+import cats.MonadThrow
+import cats.syntax.all._
+import cats.effect.ExitCode
+import org.typelevel.log4cats.Logger
+import org.scalasteward.core.io.FileAlg
+import org.scalasteward.core.repoconfig.RepoConfigAlg
+
+import ValidateRepoConfigAlg._
+import io.circe.{CursorOp, DecodingFailure, ParsingFailure}
+
+final class ValidateRepoConfigAlg[F[_]](implicit
+    fileAlg: FileAlg[F],
+    logger: Logger[F],
+    monadThrowF: MonadThrow[F]
```

Comment:
```suggestion
    F: MonadThrow[F]
```
Naming convention in this repo.